first_hour_on_vwb/working_with_data.ipynb

```diff
@@ -0,0 +1,577 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Reading data from BigQuery\n",
+    "<a id='read-from-bq'></a>\n",
+    "\n",
+    "There are many ways to interact with your referenced resource data in a Verily Workbench cloud environment. This notebook provides examples of several options, each of which are appropriate for certain use cases."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using cell magics\n",
+    "<a id='cell-magics'></a>\n",
+    "\n",
+    "For troubleshooting, demos and developing new queries, it's useful to leverage the [IPython magics for BigQuery](https://cloud.google.com/python/docs/reference/bigquery/latest/magics), several examples of which are provided below.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "<b>Note:</b> \n",
+    "    The examples in this sub-section utilize data from the 1000 Genomes data collection. Please add the BigQuery tables from that data collection to your current workspace before running the cells below. Instructions for adding a data collection to your workspace can be found <a href=\"https://support.workbench.verily.com/docs/how_to_guides/work_with_data/#add-a-data-collection\">here</a>.</div>\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Getting summary stats\n",
+    "\n",
+    "<a id='inspect-bq-stats'></a>\n",
+    "\n",
+    "To get summary statistics and visualizations for all the columns of a BigQuery table, you can use the `%bigquery_stats` cell magic. Run the cell below to view summary stats for all the table columns in the pedigree table of the 1000 Genomes dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%bigquery_stats bigquery-public-data.human_genome_variants.1000_genomes_pedigree"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### Querying with cell magics\n",
+    "\n",
+    "Run the cell below to total the number of distinct families represented in the 1000 Genomes dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%bigquery\n",
+    "\n",
+    "SELECT\n",
+    "    COUNT(DISTINCT Maternal_ID) as num_mothers,\n",
+    "    COUNT(DISTINCT Paternal_ID) as num_fathers,\n",
+    "    COUNT(DISTINCT Individual_ID) as num_individuals,\n",
+    "FROM `bigquery-public-data.`;"
```

Comment:
This doesn't work - it looks like you're missing the dataset/table

Reply:
fixed!